src/backend/TrafficCourts/Common/Models/Statute.cs

```diff
@@ -1,3 +1,4 @@
 ﻿namespace TrafficCourts.Common.Models;
 
-public record Statute(decimal Code, string Act, string Section, string Description);
+public record Statute(string StatId, string ActCd, string StatSectionTxt, string StatSubSectionTxt,
```

Comment:
Can we map to our cleaner names like

public record Statute( string Id, string ActCode, string SectionText, string SubsectionText, string ParagraphText, string SubparagraphText, string Code, string ShortDescriptionText, string DescriptionText );

Reply:
Now we have a mapper and a DTO class in Oracle Data API to map statute JSON from ORDS to a StatuteDTO utilizing Mapstruct library to save the object with cleaner names as you mentioned above for the use of other projects.